src/stories/components/delegate-summary/delegate-summary.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import DelegateSummary from './delegate-summary';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/Delegate/DelegateSummary',
+  component: DelegateSummary,
+  parameters: {
+    layout: 'fullscreen',
+    chromatic: {
+      viewports: [1440],
+      pauseAnimationAtEnd: true,
+    },
+  },
+} as ComponentMeta<typeof DelegateSummary>;
+
+const variantsArgs = [
+  {
+    code: 'del',
+  },
+];
+
+export const [[Banner, BannerDarkMode]] = createThemeModeVariants(DelegateSummary, variantsArgs);
```

Comment:
You should rename the story variants as this is not a banner. If it has only 2 variants you could use just `LightMode` and `DarkMode`

Reply:
done